### ISSUE TITLE
How can I get the recommendation result ?

### ISSUE THREAD

**Issue by sam-choi** [ardiya/Recommender-System-Tensorflow#1]
When I follow the comment "trainAE.py" and "evaluateAE.py", I can get the rmse value!
I have question , Can I get the recommendation result by using the recommender ?
An example, When I give the recommendation command in crab, "crab" give the movie recommendation like the following.

recommender.recommend(5)
[(5, 3.3477895267131013), (1, 2.8572508984333034), (6, 2.4473604699719846)]

Is there any method ??

**Comment by ardiya** [comment]
I am not on my computer, so I can only give you some advice.

You can change the X into **tf.placeholder(tf.float32, your_shape)**.
Then, feed the network using your own data using **sess.run(logits, feed_dict={X:your_data})** just like the tensorflow tutorial https://www.tensorflow.org/versions/r0.11/tutorials/mnist/beginners/index.html

I can help you with full code tomorrow, though

**Comment by sam-choi** [comment]
I need your help. I am a beginner and studying the recommender. 
I have interested in this project and result(RMSE  value ).
I want to get the recommendation result by this recommender. 
Thank you very much!

**Comment by ardiya** [comment]
Bad news, when I code, I found a mistake in the implementation of this loss function ![!loss function](https://camo.githubusercontent.com/b34884d04e0585379ea44ba9d1c150fd7dc670df/687474703a2f2f666c6f7269616e2d73747275622e636f6d2f746d702f4441454c6f73732d736d616c6c2e706e67)
So, all those RMSE numbers are incorrect. I need time to fix it, since I'm also a beginner.
Meanwhile, check out this project, it is also about recommender system and autoencoder https://github.com/henry0312/CDAE
